### PR TITLE
Update Dist_ntj221.py

### PR DESCRIPTION
**Pull request by Nojoson**

### CODE REVIEW

**Review by cranmer — CHANGES_REQUESTED** [review]
You need to delete this file or it will cause conflicts with others. 
Please delete file. Commit. and then push your changes to GitHub. 
That will update your pull request.

**Comment by cranmer** [comment]
The automated test are now working.

File: distributions/Dist_ntj221.py [hunk 1]
```diff
@@ -3,19 +3,19 @@
 
 class Dist_ntj221(BaseDistribution):
 	def __init__(self):
-		self.f_max = 1
-		self.x_min = -1
+		self.f_max = 65/120
```
Comment (cranmer) [inline distributions/Dist_ntj221.py:6]: the maximum of your function x**4+7/8*x**2 occurs when x=1, right? If so, what is f_max ?

File: distributions/Dist_ntj221.py [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
+		self.x_min = 0
 		self.x_max = 1
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return np.abs(x)
+		return x**4+(7/8)*x**2
```
Comment (cranmer) [inline distributions/Dist_ntj221.py]: I didn't check this is properly normalized. Is it?

Comment (cranmer) [reply distributions/Dist_ntj221.py]: https://www.wolframalpha.com/input/?i=Integrate%5Bx**4%2B%287%2F8%29*x**2%2C%7Bx%2C0%2C1%7D%5D

Comment (Nojoson) [reply distributions/Dist_ntj221.py]: I believe I have fixed the problem a few days ago. Has it been updated yet?

File: distributions/Dist_ntj221.py [hunk 3]
```diff
@@ -10,7 +10,7 @@ def __init__(self):
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return x**4+(7/8)*x**2
+		return (120/59)*x**4+(7/8)*x**2
```
Comment (cranmer) [inline distributions/Dist_ntj221.py]: almost...
`(120/59)*(x**4+(7/8)*x**2)`

Comment (cranmer) [reply distributions/Dist_ntj221.py]: Ugh, I wrote this comment a long time ago, but I didn't submit it properly.

Comment (cranmer) [reply distributions/Dist_ntj221.py]: Just need some ()

Comment (Nojoson) [reply distributions/Dist_ntj221.py]: Ok the distribution should be all correct now.

**Comment by cranmer** [comment]
I've merged this. You can work on part 2 now, but you will need to edit `distributions/__init__.py` to include your distribution.

**Pull request merged**